akka-actor-typed/src/main/scala/akka/actor/typed/ActorSystem.scala

```diff
@@ -139,11 +139,11 @@ abstract class ActorSystem[-T] extends ActorRef[T] with Extensions { this: inter
    * Ask the system guardian of this system to create an actor from the given
    * behavior and props and with the given name. The name does not need to
    * be unique since the guardian will prefix it with a running number when
-   * creating the child actor. The timeout sets the timeout used for the [[akka.actor.typed.scaladsl.AskPattern$]]
+   * creating the child actor. The timeout sets the timeout used for the [[akka.actor.typed.scaladsl.AskPattern]]
    * invocation when asking the guardian.
    *
    * The returned Future of [[ActorRef]] may be converted into an [[ActorRef]]
-   * to which messages can immediately be sent by using the [[ActorRef$.apply[T](s*]]
+   * to which messages can immediately be sent by using the [[ActorRef.apply[T](s*]]
```

Comment:
Sure the $'s shouldn't be in these two to disambiguate between objects and traits with the same name?

Reply:
:+1: